OpenSim/Common/Object.h

```diff
@@ -596,7 +596,9 @@ class OSIMCOMMON_API Object
     /** Write this %Object into an XML file of the given name; conventionally
     the suffix to use is ".osim". This is useful for writing out a Model that
     has been created programmatically, and also very useful for testing and
-    debugging. **/
+    debugging. If object has invalid connections, then printing is aborted.
+    You can override this behavior by calling Object::setDebugLevel(1) prior
+    to printing. **/
```

Comment:
I like below how it's precise that it's `if _debugLevel is greater than 0`, but I see why you want to give the full call here. Perhaps to do both: `You can override this behavior by setting the debug level to at least 1 (e.g., Object::setDebugLevel(1)) ...`